traefik2-loadbalancer/manifest.yaml

```diff
@@ -1,6 +1,6 @@
 ---
-name: Traefik-v2-loadbalancer
-title: Traefik v2 (LoadBalancer)
```

Comment:
This title can stay the same. Only the name needs to match the folder version

Reply:
pushed